api/client/threads_service_client_test.go

```diff
@@ -0,0 +1,102 @@
+package client
```

Comment:
This file tests registered users' ability to interact with the lower-level service API.